Base/Common.xcconfig

```diff
@@ -10,6 +10,9 @@ ALWAYS_SEARCH_USER_PATHS = NO
 // Architectures to build
 ARCHS = $(ARCHS_STANDARD)
 
+// Check for Grand Central Dispatch idioms that may lead to poor performance.
+CLANG_ANALYZER_GCD_PERFORMANCE = YES
```

Comment:
In general, I'm in favor of this addition, though I did find a conversation about this setting producing warnings with no clear fixes: https://twitter.com/steipete/status/1005893961926479872
Having not encountered such issues myself, I'm not sure if that's reason enough to avoid enabling the build setting by default.

Reply:
Yeah, good call. 
We did see 2 analyzer warnings pop up because of this as well. (semaphore usage to make async call sync). But yeah, they're kind of right :-P 
My hunch would be to enable in xcconfigs -> it's easy for users to disable per project, while also disable them on a per file basis with
```
#ifndef __clang_analyzer__

// Code not to be analyzed

#endif
```
via https://stackoverflow.com/questions/5806101/is-it-possible-to-suppress-xcode-4-static-analyzer-warnings